src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Form\Admin\Sell\Product\Image;
+
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\Form\Util\StringUtil;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * This form type integrates a Dropzone used to manage images, the twig templates is a simple div
+ * with data attributes, it needs javascript from the Dropzone.vue component to work.
+ *
+ * This form type though should be used to pass any configuration to the javascript components, like
+ * translations, locales, or productId (maybe even urls could be configurable).
+ */
+class ImageDropzoneType extends TranslatorAwareType
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'translations' => [
+                'window.selectAll' => $this->trans('Select all', 'Admin.Actions'),
+                'window.settingsUpdated' => $this->trans('Settings udpated', 'Admin.Actions'),
+                'window.imageReplaced' => $this->trans('Image replaced', 'Admin.Actions'),
+                'window.unselectAll' => $this->trans('Unselect all', 'Admin.Actions'),
+                'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
+                'window.cantDisableCover' => $this->trans('You cannot disable the cover image.', 'Admin.Actions'),
```

Comment:
I find this wording not very clear. Here is another suggestion: 
'Choosing another image as the cover will automatically uncheck this box. '